scripts/run_performance_tests.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+##########################################################################
+
+# INSTRUCTIONS:
+#
+# Run this script from the oppia root folder:
+#   bash scripts/run_performance_tests.sh
+# 
+# The root folder MUST be named 'oppia'.
+
+function cleanup {
+  # Send a kill signal to the dev server.
+  kill `ps aux | grep "[Dd]ev_appserver.py --host=0.0.0.0 --port=9501" | awk '{print $2}'`
+
+  # Wait for the servers to go down; suppress "connection refused" error output
+  # from nc since that is exactly what we are expecting to happen.
+  while ( nc -vz localhost 9501 >/dev/null 2>&1 ); do sleep 1; done
+
+  echo Done!
+}
+
+if [ -z "$BASH_VERSION" ]
+then
+  echo ""
+  echo "  Please run me using bash: "
+  echo ""
+  echo "     bash $0"
+  echo ""
+  return 1
+fi
+
+set -e
+source $(dirname $0)/setup.sh || exit 1
+source $(dirname $0)/setup_gae.sh || exit 1
+
+export DEFAULT_SKIP_INSTALLING_THIRD_PARTY_LIBS=false
+export DEFAULT_RUN_MINIFIED_TESTS=false
+maybeInstallDependencies "$@"
+
+if ( nc -vz localhost 8181 ); then
```

Comment:
This is for checking whether a dev server is already running on 8181. Although, I do not see any issues with running 2 dev servers on different ports, the e2e tests script had this check, so I kept it there. Is there any reason in particular for this?